ocrd_network/ocrd_network/logging.py

```diff
@@ -0,0 +1,48 @@
+from os import makedirs
+from os.path import join
+from ocrd_utils import safe_filename
+
+OCRD_NETWORK_MODULES = [
+    "mets_servers",
+    "processing_jobs",
+    "processing_servers",
+    "processing_workers",
+    "processor_servers"
+]
+
+
+def get_root_logging_dir(module_name: str) -> str:
+    if module_name not in OCRD_NETWORK_MODULES:
+        raise ValueError(f"Invalid module name: {module_name}, should be one of: {OCRD_NETWORK_MODULES}")
+    # TODO: Utilize env variable to set the root
+    module_log_dir = join("/tmp/ocrd_network_logs", module_name)
+    makedirs(name=module_log_dir, exist_ok=True)
```

Comment:
I would use [pathlib](https://docs.python.org/3/library/pathlib.html) for all new fs-related code.
```suggestion
    module_log_dir = Path(tempfile.gettempdir(), "ocrd_network_logs", module_name)
    module_log_dir.mkdir(exist_ok=True)
```

Reply:
Resolved and using Path, however, I am still not using `tempfile.gettempdir()`. For the logging root, I don't think it's a good idea to use a temporary directory. Especially, for the dockerized version where volume mapping should be persistent in the docker compose file. Let me know if you have a better idea.

Reply:
Resolved in https://github.com/OCR-D/core/pull/1111/commits/464f8eaf8c2c696668699a880f885dea14847f1b